projects/1.16/assets/xplosives-mod/xplosives/lang/zh_cn.json

```
@@ -0,0 +1,25 @@
+{
+  "block.xplosives.arrow_bomb": "弓箭炸弹",
+  "block.xplosives.bonemeal_bomb": "骨粉炸弹",
+  "block.xplosives.cluster_bomb": "集束炸弹",
+  "block.xplosives.fireblast_tnt": "火焰弹TNT",
+  "block.xplosives.hydrogen_bomb": "氢弹",
+  "block.xplosives.incendiary_bomb": "燃烧炸弹",
+  "block.xplosives.lightning_conductor": "雷击炸弹",
+  "block.xplosives.packed_tnt": "压缩TNT",
+  "block.xplosives.physics_bomb": "物理炸弹",
+  "block.xplosives.snowball_bomb": "雪球炸弹",
+  "block.xplosives.tipped_arrow_bomb": "药箭炸弹",
+  "block.xplosives.detonator": "遥控雷管",
+  "block.xplosives.gunpowder_barrel": "火药桶",
+  "block.xplosives.torch_bomb": "火把炸弹",
+  "block.xplosives.soul_torch_bomb": "灵魂火把炸弹",
+  "block.xplosives.mini_tnt": "迷你TNT",
+  "item.xplosives.detonator_remote": "遥控雷管引爆器",
+  "item.xplosives.black_box": "炸弹发射器",
```

这啥

---

[Black Box](https://wiki.teamfortress.com/wiki/Black_Box/zh-hans)是军团要塞2的火箭筒，模组内可以发射任意炸药后引爆……但考虑到国内并没有多少玩军团要塞2的所以意译为炸弹发射器。翻译成军团要塞2的黑匣子的话我觉得会导致歧义

---

```suggestion
  "item.xplosives.black_box": "黑匣子",
```
那么，显然，不玩军团要塞的英语母语者也看不懂 Black Box，无需考虑这个问题。